Source/loadsave.cpp

```diff
@@ -1442,14 +1442,14 @@ const int HellfireItemSaveSize = 372;
 
 void RemoveInvalidItem(Item &item)
 {
-	bool isInvalid = !IsItemAvailable(item.IDidx) || !IsUniqueAvailable(item._iUid);
+	bool isInvalid = item.IDidx > IDI_LAST || !IsItemAvailable(item.IDidx) || !IsUniqueAvailable(item._iUid);

```

Comment:
This doesn't use the same check for negative values as `UnPackItem` does.

Reply:
Yeah I got a suggestion for changing the other one, but didn't get around to change this one manually yet :)